reference/ams-configuration.md

```diff
@@ -30,22 +30,46 @@ AMS provides various configuration items to customize its behavior. The followin
 
 Some features can be conditionally enabled via AMS.
 
+All feature flags can either be globally configured for all containers in the AMS configuration
+
+    amc config set container.features foo,bar
+
+or per application in the manifest
+
+    name: my-app
+    instance-type: a4.3
+    features: ["foo", "bar"]
+
 #### System UI
 
 *since 1.10.2*
 
-By default Anbox hides the Android system UI when an application is running in foreground mode. In some use cases it's however required to have the system UI available for navigation purposes. The system UI can be enabled via feature flag, e.g.:
-
-    $ amc config set container.features "enable_system_ui"
+By default Anbox hides the Android system UI when an application is running in foreground mode. In some use cases it's however required to have the system UI available for navigation purposes. This can be enabled with the `enable_system_ui` feature flag.
 
 The feature flag will be considered by all new launched containers once set.
 
 #### Virtual Keyboard
 
 *since 1.9.0*
 
-The Android virtual keyboard is disabled by default but can be enabled by adding the `enable_virtual_keyboard` feature, e.g.:
+The Android virtual keyboard is disabled by default but can be enabled with the `enable_virtual_keyboard` feature flag
+
+For the feature to be considered, applications need to be manually updated as changes to allow the feature to work are only applied during the [application bootstrap process](https://discourse.ubuntu.com/t/managing-applications/17760#bootstrap).
+
+#### WiFi
+
+*since 1.12.0*
 
-    $ amc config set container.features "enable_virtual_keyboard"
+WiFi support can be optionally enabled. Anbox will then setup a virtual WiFi device which sits on top of an ethernet connection and simulates a real WiFi connection.
 
-Note that you need to manually update the existing applications for the change to take effect. After the application gets updated,  containers that are launched from the latest application version will have the virtual keyboard enabled by default.
+The feature flag will be considered by all new launched containers once set.
+
+### Android Reboot
+
+*since 1.12.0*
+
+By default Android is disallowed to reboot. With the `allow_android_reboot` feature flag this can be allowed.
+
+Please note that you have to disable the  [watchdog](https://discourse.ubuntu.com/t/application-manifest/24197#watchdog) if reboots are allowed.
+
+The feature flag will be considered by all new launched containers once set.
```

Comment:
```suggestion
By default, Android is not allowed to reboot. With the `allow_android_reboot` feature flag, this can be allowed.

Note that you must disable the [watchdog](https://discourse.ubuntu.com/t/application-manifest/24197#watchdog) if reboots are allowed.

The feature flag will be considered by all newly launched containers once set.
```